### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by McLive**

### CODE REVIEW

File: styles/language/German/language.php [hunk 1]
```diff
@@ -51,31 +52,31 @@
...
+	'announcement_content' => 'Ankündigungsinhalt',
+	'announcement_location' => 'Ankündigungsposition',
+	'announcement_can_close' => 'Kann die Ankündigung schließen?',
 	'announcement_permissions' => 'Hinweisrechte',
-	'no_announcements' => 'Noch keine Hinweise vorhanden.',
-	'confirm_cancel_announcement' => 'Willst du wirklich diesen Hinweis löschen?',
-	'announcement_location_help' => 'Strg-Klick um mehrere Seiten zu wählen',
+	'no_announcements' => 'Noch keine Ankündigungen vorhanden.',
+	'confirm_cancel_announcement' => 'Willst du diese Ankündigung wirklich stornieren?',
+	'announcement_location_help' => 'Strg-Klick, um mehrere Seiten zu wählen',
```
Comment (Cl1608Ho) [inline styles/language/German/language.php]: No comma here.

File: styles/language/German/language.php [hunk 2]
```diff
@@ -51,31 +52,31 @@
...
+	'create_announcement' => 'Ankündigung erstellen',
+	'announcement_content' => 'Ankündigungsinhalt',
+	'announcement_location' => 'Ankündigungsposition',
+	'announcement_can_close' => 'Kann die Ankündigung schließen?',
 	'announcement_permissions' => 'Hinweisrechte',
-	'no_announcements' => 'Noch keine Hinweise vorhanden.',
-	'confirm_cancel_announcement' => 'Willst du wirklich diesen Hinweis löschen?',
-	'announcement_location_help' => 'Strg-Klick um mehrere Seiten zu wählen',
+	'no_announcements' => 'Noch keine Ankündigungen vorhanden.',
+	'confirm_cancel_announcement' => 'Willst du diese Ankündigung wirklich stornieren?',
```
Comment (Cl1608Ho) [inline styles/language/German/language.php]: Are you sure you want to use "stornieren"? I would keep "löschen"

File: styles/language/German/language.php [hunk 3]
```diff
@@ -84,33 +85,33 @@
...
 	'use_gmail_help' => 'Diese Funktion ist nur möglich, wenn die PHP mail() Funktion deaktiviert ist. Wenn du nicht GoogleMail verwendest, wird SMTP verwendet. Beide Wege müssen manuell in core/mail.php gesetzt werden.',
 	'enable_mail_verification' => 'Aktiviere Email verifikation?',
 	'enable_email_verification_help' => 'Wenn dies aktiviert ist, müssen sich neue Nutzer per Email freischalten.',
-	'explain_email_settings' => 'The following is required if the "Use PHP mail() function" option is <strong>disabled</strong>. You can find documentation on these settings <a href="https://github.com/NamelessMC/Nameless/wiki/Setting-up-Gmail-or-SMTP-with-Nameless" target="_blank">on our wiki</a>.',
-	'email_config_not_writable' => 'Bitte gibt <strong>core/email.php</strong> schreib-Rechte. Wir können auf die Datei nicht zugreifen.',
+	'explain_email_settings' => 'Die folgende Einstellung wird benötigt, wenn die "Use PHP mail()" Option <strong>deaktiviert</strong> ist. Die Dokumentation dafür findest du <a href="https://github.com/NamelessMC/Nameless/wiki/Setting-up-Gmail-or-SMTP-with-Nameless" target="_blank">im Wiki</a>.',
```
Comment (Cl1608Ho) [inline styles/language/German/language.php]: Please translate "Use php mail()" as above.

File: styles/language/German/language.php [hunk 4]
```diff
@@ -489,19 +490,19 @@
...
 	'invalid_tfa' => 'Ungültiger Code, versuche es erneut.',
-	'tfa_successful' => '2Step Authentication erfolgreich eingerichtet.',
-	'confirm_tfa_disable' => 'Willst du die 2Step Authentication deaktivieren?',
-	'tfa_disabled' => 'Du hast die 2Step Authentication deaktiviert.',
-	'tfa_enter_email_code' => 'Gib den dir per Email zugeschickten Code ein:',
-	'tfa_email_contents' => 'Jemand hat sich versucht mit deinem Konto anzumelden, warst dies du, gib folgenden Code ein:',
+	'tfa_successful' => 'Zwei-Faktor-Authentifizierung erfolgreich eingerichtet.',
+	'confirm_tfa_disable' => 'Willst du die Zwei-Faktor-Authentifizierung deaktivieren?',
+	'tfa_disabled' => 'Du hast die Zwei-Faktor-Authentifizierung deaktiviert.',
+	'tfa_enter_email_code' => 'Gib den dir per E-Mail zugeschickten Code ein:',
```
Comment (Cl1608Ho) [inline styles/language/German/language.php]: Can you change it like "Gib den Code ein, der dir per Email zugeschickt wurde"

File: styles/language/German/language.php [hunk 5]
```diff
@@ -569,10 +570,10 @@
 	'follow' => 'Folgen',
 	'unfollow' => 'Nicht mehr Folgen',
 	'name_history' => 'Name History',
-	'changed_name_to' => 'Hat am {y}  seinen Namen zu {x} geändert.', // Don't replace {x} or {y}
-	'original_name' => 'Original Name:',
+	'changed_name_to' => 'Hat am {y} seinen Namen zu {x} geändert.', // Don't replace {x} or {y}
+	'original_name' => 'Originaler Name:',
```
Comment (Cl1608Ho) [inline styles/language/German/language.php]: I'd prefer "Voriger Name"

**Comment by Cl1608Ho** [comment]
I know I could just wait for this to be merged and open a pr myself, but IMO this is better ;)

File: styles/language/German/language.php (hunk 2 above)
Comment (McLive) [inline styles/language/German/language.php]: I'd call it like that, because "löschen" is better for "delete_announcement"

**Comment by BukkitTNT** [comment]
There are many mistakes in the translation. (I am from Germany, German is my Main-Language).

**Comment by McLive** [comment]
Indeed. I tried to fix some of them ;)

**Comment by samerton** [comment]
Thanks for this

**Pull request merged**